xc-tool/build.rs

```diff
@@ -0,0 +1,107 @@
+use std::{env, fs, path::PathBuf, path::Path};
+
+use ethers::contract::Abigen;
+use ethers::solc::{remappings::Remapping, Artifact, Project, ProjectPathsConfig, Solc};
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let pyth_evm_src_root =
+        env::var("PYTH_XC_TOOL_EVM_SRC_ROOT").unwrap_or("../ethereum/".to_string());
```

Comment:
nit: not for now, but for later, it's generally easier to use a config crate such as https://crates.io/crates/config instead of ad-hod environment variables

Reply:
I am not a fan of elaborate build.rs dependencies, I think we should consider that when we get closer to 10-20 variables for the build.rs